README.md

```diff
@@ -32,9 +33,51 @@ You should see `Git LFS initialized` or a similar message. **Important**: if you
 
 Now, you can download the repository:
 
-- `git clone --recursive https://github.com/JoshParnell/ltheory.git ltheory`
+- `git clone https://github.com/Limit-Theory-Redux/ltheory.git`
 
-## Compiling
+## Compiling (Option 1: Precompiled Bin)
```

Comment:
I think it would be clearer if this section was laid out in this way, making it clear to users that this is one section with two choices (as subsections):

```
## Setup libphx

Next, we will need to get the engine ready to run Lua applications. There are two options, downloading a precompiled binary package, or compiling libphx manually.

### Option 1: Using precompiled binaries.

.....

### Option 2: Compiling libphx manually

....

```